conjure-java-core/src/test/resources/test/api/AsyncMarkers.java.undertow

```diff
@@ -1,13 +1,19 @@
 package test.api;
 
+import com.google.common.util.concurrent.ListenableFuture;
 import javax.annotation.Generated;
 
 @Generated("com.palantir.conjure.java.services.UndertowServiceInterfaceGenerator")
 public interface AsyncMarkers {
     /**
-     * @apiNote {@code GET /async}
+     * @apiNote {@code GET /async/marker}
      */
-    String async();
+    String asyncMarker();
```

Comment:
shouldn't this also be async?

Reply:
Nope, this is from a test case without `experimentalUndertowAsyncMarkers`. The `.undertow.async` output reflects `experimentalUndertowAsyncMarkers(true)`. Note that the tag variation does not rely on the experimental markers flag.